src/headers/writer.rs

```diff
@@ -14,44 +14,31 @@ use super::MAX_LINE_LEN;
 pub struct EmailWriter<'a> {
     writer: &'a mut dyn Write,
     line_len: usize,
-    write_space_on_next_write: bool,
+    spaces: usize,
+    write_decorative_space_on_next_line: bool,
```

Comment:
Maybe `optional_breakpoint` may be a better name.

Or I wonder what the code would look like if it was just called `breakable` and it was always set to `true` when setting `spaces` to non-zero. That may remove some conditionals but IDK if it is better overall.

Either way it probably makes sense to add docs to these fields. I see that they were sort of documented in `new` but I think struct documentation is a better centralized place and may be shown by tooling or private-enabled rustdoc builds.

Reply:
Changed the name. I'll send the docs in a followup PR